src/main/java/com/kramekk1/medicalClinic/PatientService.java

```diff
@@ -40,4 +40,10 @@ public void deletePatient(String email) {
         patientRepository.deletePatient(email);
     }
 
+    public void editPasswordByEmail(String email, String newPassword) {
+        Optional<Patient> patient = patientRepository.findPatientByEmail(email);
+        patient.ifPresent(PatientValidator::validateEmailNotNull);
+        PatientValidator.validateIncorrectEmail(patient);
```

Comment:
to jest edycja hasła w bazie danych nie powinienes miec pacjenta o mailu ktory jest null wiecwydaje mi sie ze nie ma tutaj potrzeby dodawać tej walidacji.